src/witan/gateway/queries/workspace.clj

```diff
@@ -32,21 +31,21 @@
   [])
 
 (defn get-workspaces-by-owner
-  [d _ owner]
+  [u d owner]
```

Comment:
I understand what those are but not a fan of single-letter variables for cognitive effort (I imagine you might change this for final version)